### PR TITLE
SB-680: Create a hello-strongbox-nuget example (using Mono)

### PR DESCRIPTION
**Pull request by carlspring**

### CODE REVIEW

**Comment by carlspring** [comment]
I think it actually would probably be a better idea to call the example `hello-strongbox-nuget-mono`.

File: hello-strongbox-nuget/Mono.HelloWorld.nuspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata>
+		<id>Strongbox.Nuget.HelloWorld</id>
```
Comment (carlspring) [inline hello-strongbox-nuget/Mono.HelloWorld.nuspec]: @sbespalov,

Could you please change this to `Org.Carlspring.Strongbox.Examples.Nuget.Mono`? (I believe this is the standard naming convention under Nuget).

File: hello-strongbox-nuget/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    mono --runtime=v4.0.30319 nuget.exe pack ./Mono.HelloWorld.nuspec
+    
+The output should be like follows:
+
+    carlspring@linux-70e2:/home/carlspring/strongbox-examples/hello-strongbox-nuget> mono --runtime=v4.0.30319 nuget.exe pack ./Mono.HelloWorld.nuspec 
+    Attempting to build package from 'Mono.HelloWorld.nuspec'.
+    Successfully created package '/home/carlspring/strongbox-examples/hello-strongbox-nuget/Strongbox.Nuget.HelloWorld.1.0.nupkg'.
+
+# How to push NuGet package into Carlspring repository
```
Comment (carlspring) [inline hello-strongbox-nuget/README.md]: `# How to push NuGet package into Strongbox repository`

File: hello-strongbox-nuget/Mono.HelloWorld.nuspec (hunk 1 above)
Comment (sbespalov) [reply hello-strongbox-nuget/Mono.HelloWorld.nuspec]: done

File: hello-strongbox-nuget/README.md (hunk 2 above)
Comment (sbespalov) [reply hello-strongbox-nuget/README.md]: done

File: hello-strongbox-nuget/Mono.HelloWorld.nuspec (hunk 1 above)
Comment (sbespalov) [inline hello-strongbox-nuget/Mono.HelloWorld.nuspec]: done

File: hello-strongbox-nuget/README.md (hunk 2 above)
Comment (sbespalov) [inline hello-strongbox-nuget/README.md]: done

File: hello-strongbox-nuget-mono/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+This is an example of how to use the Strongbox artifact repository manager with NuGet.
+
+## Pre-requisites
+The following is a minimal configuration in order to be able to use this tutorial:
+
+* `Linux` OS
```
Comment (carlspring) [inline hello-strongbox-nuget-mono/README.md:6]: Mono can run on Windows as well, so I don't think this is a requirement.

**Comment by carlspring** [comment]
@sbespalov ,

I think a brief explanation of the nuspec file will also be quite useful. What it is, what it's used for, what the basic things it contains are, etc, etc... :)

File: hello-strongbox-nuget-mono/README.md (hunk 3 above)
Comment (sbespalov) [reply hello-strongbox-nuget-mono/README.md:6]: You right about Mono but I am not sure about all commands from this tutorial that they can run on Windows as is.

**Comment by sbespalov** [comment]
I think we can merge it as [PR](https://github.com/strongbox/strongbox/pull/180) was merged. We can add some additional features separately.

**Comment by carlspring** [comment]
I agree! Thanks for your contribution!

**Pull request merged**